content/applications/sales/sales/products_prices/products/variants.rst

```diff
@@ -1,225 +1,178 @@
+Products`, and then, either select a product to add variants or create a new one by tapping
+:guilabel:`Create`.
 
-Now, click on the product you wish to add variants to.
+The smart button at the top of the product template indicates the number of currently configured
+variants on this product.
 
-In the product page, a new tab called Variants has appeared. The number
-in purple written on top is the number of variants this product
-currently has. To add new variants, click on the tile. In the new
-window, click on **Create**.
-
-In **Attributes**, click on the rolldown menu and select the type of
-variance you wish to add. If the variant does not yet exist, you can
-create it on the fly by clicking on Create and edit…
-
-.. image:: variants/variant02.png
-   :align: center
-
-In the Attributes window, the **Value** field is the description of the
-attribute such as Green, Plastic or 32GB. The **Attribute** field is the
-type of variant such as Color, Material or Memory.
-
-.. image:: variants/variant03.png
+.. image:: variants/variant-smart-button.png
    :align: center
+   :alt: Variants smart button.
 
-You can add a cost for the variant on the fly by adding it in the
-**Attribute Price Extra** field, or choose to modify it later. Click on
-**Save**.
+To add a new variant, click on the :guilabel:`Variants` tab, then click on :guilabel:`Add Any Line`
+to add any attributes and values.
 
-You can also add a different barcode and internal reference to the
-variant.
-
-When you have entered all the specifications of the variant, click on
-**Save**.
-
-Managing Product Variants
-=========================
-
-Introduction
-------------
-
-The examples below are all based on this product template that has two
-variant attributes :
+.. tip::
+   Similar creation processes are accessible through the Purchase, Inventory, and eCommerce
+   applications.
 
--  T-Shirt B&C
+Managing product exclusions
+---------------------------
 
-   -  Color: Red, Blue, White
+The following examples are all based on a product template that has two attributes:
 
-   -  Size: S, M, L, XL, XXL
+- T-Shirt
 
-Managing combination possibilities
-----------------------------------
+  - Color: Blue, Red, White, Black
+  - Size: S, M, L, XL, XXL
 
-By default, with the above product template, you get 15 different
-products (3 colors, 5 sizes). If the XXL size only exists for red and
-blue t-shirts, you can deactivate the white product variant.
+With the above product template, there are 15 different t-shirt variants in three different colors
+and five different sizes. If the white t-shirts are not available in the XXL size, then that
+variant can be deactivated.
 
-To do this, click on the **Variants** button, select the XXL, White
-T-shirt. From the product form, uncheck the **Active** box of the T-shirt
-White, XXL.
+To do this, go to :menuselection:`Sales --> Products --> Products` and select the relevant product.
```

Comment:
Editing so that people skimming the doc can know immediately what this paragraph is about
```suggestion
To deactivate a particular product variant, go to :menuselection:`Sales --> Products --> Products` and select the relevant product.
```